distributed/worker.py

```diff
@@ -1602,6 +1606,25 @@ def transition_waiting_ready(self, ts):
                 pdb.set_trace()
             raise
 
+    def transition_ready_waiting(self, ts, worker):
+        # TODO: do we actually want this?  or is this a bandaid for
+        # bad bookkeeping elsewhere?
```

Comment:
I'm curious.  What caused the need for this?

Reply:
I think this is actually a result of some bad bookkeeping, but I stuck it in there to both give me a place to set a breakpoint, but also to see where else things broke if there wasn't a `KeyError` in `self._transitions`.  It should not be required once I've sorted that out.